escargot.gyp

```diff
@@ -47,6 +47,7 @@
       'escargot_configs': [
         '-DESCARGOT_SMALL_CONFIG=1',
         '-DESCARGOT_USE_CUSTOM_LOGGING=ON',
+        '-DESCARGOT_USE_EXTENDED_API=ON',
```

Comment:
Escargot build option updated.
To use specific non-standard features of Escargot like Template, build option `ESCARGOT_USE_EXTENDED_API` should be set ON.